### DISCUSSION TITLE
[Tác giải bộ gõ tiếng việt] Anh có thể quay thành 1 khóa học về cách tạo ra bộ gõ tiếng việt trên linux được không anh??

### DISCUSSION THREAD

**Discussion by hoainam10th123** [BambooEngine/ibus-bamboo#431] [Ideas]
Anh có thể quay thành video cách bộ gõ hoạt động, ý tưởng để thực hiện, giải thích code. cuối cùng là cách triển khai thành package để mọi người cài đặt trên máy linux.
Em mê cái bộ gõ này lắm mà năng lực, kiến thức chưa có để làm được.
Anh quay thành khóa học mọi người mua ủng hộ, đồng thời cũng là 1 cách để duy trì dev qua các thế hệ để phát triển bộ gõ.
xin cảm ơn!

**Comment by t1ld3x** [comment]
bạn tham khảo tài liệu này nhé https://github.com/BambooEngine/ibus-bamboo/wiki/H%C6%B0%E1%BB%9Bng-d%E1%BA%ABn-c%C4%83n-b%E1%BA%A3n-l%C3%A0m-b%E1%BB%99-g%C3%B5-Ti%E1%BA%BFng-Vi%E1%BB%87t

**Comment by goatastronaut0212** [comment]
> Anh có thể quay thành video cách bộ gõ hoạt động, ý tưởng để thực hiện, giải thích code. cuối cùng là cách triển khai thành package để mọi người cài đặt trên máy linux. Em mê cái bộ gõ này lắm mà năng lực, kiến thức chưa có để làm được. Anh quay thành khóa học mọi người mua ủng hộ, đồng thời cũng là 1 cách để duy trì dev qua các thế hệ để phát triển bộ gõ. xin cảm ơn!

Cá nhân mình không thích việc có và bán 1 khóa học để hack và code 1 dự án nguồn mở lắm, mình không biết là nói trước thì có thể thực hiện được hết không. Nhưng mình về cơ bản có thể đọc và hiểu 1 số code của dự án này nếu không ai chê khả năng kém cỏi của mình thì mình sẽ thử làm 1 document cách dự án hoạt động.
